common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -161,15 +163,20 @@ def _test_return_login(self, user_is_activated=True, previous_session_timed_out=
         # required to set the login cookie (it sticks around if the main session times out):
         if not previous_session_timed_out:
             self.assertEqual(login_response.status_code, 302)
-            self.assertEqual(login_response['Location'], self.url_prefix + self.complete_url)
+            expected_url = expected_redirect_url(self.complete_url, hostname='example.none')
+            # TODO: Remove Django 1.11 upgrade shim
+            # SHIM: Get rid of this logic post-upgrade
+            if django.VERSION >= (1, 9):
+                expected_url = "{}?".format(expected_url)
+            self.assertEqual(login_response['Location'], expected_url)
```

Comment:
Not entirely sure this fix is correct - but it does fix the test.

Reply:
I agree this is weird, but wasn't able to sort out why/where it changed. I don't _think_ it should cause any issues, but it would sure be nice to know what's happening here.